lg69t/README.md

```diff
@@ -16,21 +16,22 @@ python3 -m pip install -r requirements.txt
 
 And download the latest firmware package from [Point One's Developer Portal](https://pointonenav.com/docs/).
 
-
 ## Application and GNSS
 
-To update the Application and GNSS firmware:
+To update the Application and GNSS firmware from previous Point One Firmware, you must be use the UART1(Standard COM Port for Windows, typically /dev/ttyUSB1 in Linux for P1SDK):
```

Comment:
Space after UART1, and add backticks around "Standard COM Port" and "dev/ttyUSB1" so they format and stand out since those are verbatim names/paths that will exist on the machine

Reply:
must use UART1, not "the" UART1. Space between word and (.

Reply:
also eliminate the "from previous Point One Firmware" as that's not a real requirement.